content/blog/2021/cff-best-practises.md

```diff
@@ -0,0 +1,71 @@
+Title: Best Practises to look out for when submitting talks
+Date: 2021-04-07 17:09
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2021
+Slug: cfp-best-practises
+Summary: PyCon India 2021 is looking for talks to be presented at the conference. Read on to know when submitting a proposal how you too can make your talk awesome. 
+
+PyCon India is the largest gathering of Pythonistas in India for the Python programming language. The 13th edition of PyCon India will be taking place online from 17th September to 20th September 2021. So, whether you are someone submitting a proposal for talks for the first time or just want to share what you have been up to with Python for a long time. This is YOUR time to shine!
```

Comment:
```suggestion
[**PyCon India**](https://in.pycon.org/2021/) is the largest gathering of Pythonistas in India for the Python programming language. The 13th edition of PyCon India will be taking place online from 17th September to 20th September 2021. So, whether you are someone submitting a proposal for talks, or workshops, for the first time or just want to share what you have been up to with Python for a long time. This is YOUR time to shine!
```

Reply:
There is no mention of workshop in this blog, we will do a workshop blog seperately. Check last year archive.